City-of-Helsinki/hel-fi-drupal-grants
# fix: AU-1453: Configure leiri-excel attachment checkboxes

# [AU-1453](https://helsinkisolutionoffice.atlassian.net/browse/AU-1453)
## What was done


* Hide checkboxes so related attachment field is always required.

## How to install and test

* Make sure your instance is up and running on correct branch.
  * `git checkout AU-1453-attachment-config`
  * `drush gwi --force` inside the virtual machine to update webform
  * Test the form in https://hel-fi-drupal-grant-applications.docker.so/en/form/nuorlomaleir 


[AU-1453]: https://helsinkisolutionoffice.atlassian.net/browse/AU-1453?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
